src/Elcodi/Fixtures/DataFixtures/ORM/AdminUser/AdminUserData.php

```diff
@@ -42,7 +42,6 @@ public function load(ObjectManager $manager)
             ->container
             ->get('elcodi.factory.admin_user')
             ->create()
-            ->setUsername('johnwayne')
```

Comment:
Doesn't this affect the symfony debug toolbar (the area where it displays the logged user) ?